bin/reth/src/node/mod.rs

```diff
@@ -149,13 +149,10 @@ impl Command {
         let _secret = self.rpc.jwt_secret();
```

Comment:
@rkrasiuk i think in one of your PRs we need to actually spawn the engine api here